TestCases/compliance-level-3/0054-feel-even-function/0054-feel-even-function.dmn

```diff
@@ -0,0 +1,158 @@
+        <description>Tests FEEL expression: 'even(-2)' and expects result: 'true (boolean)'</description>
+        <question>Result of FEEL expression 'even(-2)'?</question>
+        <allowedAnswers>true (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision002"/>
+        <literalExpression>
+            <text>even(-2)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision002_a" id="_decision002_a">
+        <description>Tests FEEL expression: 'even(-1)' and expects result: 'false (number)'</description>
+        <question>Result of FEEL expression 'even(-1)'?</question>
+        <allowedAnswers>false (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision002_a"/>
+        <literalExpression>
+            <text>even(-1)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision003" id="_decision003">
+        <description>Tests FEEL expression: 'even(0)' and expects result: '0 (number)'</description>
+        <question>Result of FEEL expression 'even(0)'?</question>
+        <allowedAnswers>0 (number)</allowedAnswers>
+        <variable typeRef="boolean" name="decision003"/>
+        <literalExpression>
+            <text>even(0)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision004" id="_decision004">
+        <description>Tests FEEL expression: 'even()' and expects result: 'null (number)'</description>
+        <question>Result of FEEL expression 'even()'?</question>
+        <allowedAnswers>null (number)</allowedAnswers>
+        <variable typeRef="boolean" name="decision004"/>
+        <literalExpression>
+            <text>even()</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision005" id="_decision005">
+        <description>Tests FEEL expression: 'even(4,4)' and expects result: 'null (number)'</description>
+        <question>Result of FEEL expression 'even(4,4)'?</question>
+        <allowedAnswers>null (number)</allowedAnswers>
+        <variable typeRef="boolean" name="decision005"/>
+        <literalExpression>
+            <text>even(4,4)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision006" id="_decision006">
+        <description>Tests FEEL expression: 'even(number:4)' and expects result: '2 (number)'</description>
+        <question>Result of FEEL expression 'even(number:4)'?</question>
+        <allowedAnswers>2 (number)</allowedAnswers>
+        <variable typeRef="boolean" name="decision006"/>
+        <literalExpression>
+            <text>even(number:4)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision007" id="_decision007">
+        <description>Tests FEEL expression: 'even(n:4)' and expects result: 'null (number)'</description>
+        <question>Result of FEEL expression 'even(n:4)'?</question>
+        <allowedAnswers>2 (number)</allowedAnswers>
```

Comment:
from ` <allowedAnswers>2 (number)</allowedAnswers>`
to ` <allowedAnswers>null (boolean)</allowedAnswers>`

Reply:
@tarilabs amended.